core-web/libs/block-editor/src/lib/nodes/contentlet-block/contentlet-block.node.ts

```diff
@@ -39,7 +39,18 @@ export const ContentletBlock = (injector: Injector): Node<ContentletBlockOptions
         },
 
         renderHTML({ HTMLAttributes }): DOMOutputSpec {
-            return ['dotcms-contentlet-block', mergeAttributes(HTMLAttributes)];
+            let img = [];
+            if (HTMLAttributes.data.hasTitleImage) {
+                img = ['img', { src: HTMLAttributes.data.image }];
+            }
+
+            return [
+                'div',
+                ['h3', { class: HTMLAttributes.data.title }, HTMLAttributes.data.title],
+                ['div', HTMLAttributes.data.identifier],
+                img,
+                ['div', {}, HTMLAttributes.data.language]
+            ];
```

Comment:
```suggestion
            // The `.. || {}` is to prevent errors when accessing its properties
            const {hasTitleImage,  title, identifier, language, image} = HTMLAttributes?.data || {};
            const img = hasTitleImage ? ['img', { src: image }] : [];

            return [
                'div',
                ['h3', { class: title }, title],
                ['div', identifier],
                img,
                ['div', {}, language]
            ];
```